pages/installation/os/ubuntu.rst

```diff
@@ -19,40 +19,39 @@ Taking a minimal server setup as base will need this additional packages::
 MongoDB
 -------
 
-The Version included in Ubuntu 16.04 LTS can be used together with Graylog 2.0.0 and higher::
+The Version included in Ubuntu 16.04 LTS can be used together with Graylog 2.3.x and higher::
 
   $ sudo apt-get install mongodb-server
 
 
 Elasticsearch
 -------------
 
-Graylog 2.0.0 and higher requires Elasticsearch 2.x, so we took the installation instructions from `the Elasticsearch installation guide <https://www.elastic.co/guide/en/elasticsearch/reference/2.3/setup-repositories.html#_apt>`__::
+Graylog 2.3.x can be used with Elasticsearch 5.x, please follow the installation instructions from `the Elasticsearch installation guide <https://www.elastic.co/guide/en/elasticsearch/reference/current/deb.html>`__::
```

Comment:
The link to the Elasticsearch documentation should use a specific version:

https://www.elastic.co/guide/en/elasticsearch/reference/5.4/deb.html